input_manager/include/core/EditorInputManager.hpp

```diff
@@ -86,22 +89,14 @@ class EditorInputManager final : protected InputManagerImplementation {
                     }
 
                     if (im.getValue(rcbe::core::KeyboardEventType::arrow_up)) {
-                        auto pitch = c->getPitch();
-                        if (static_cast<double>(pitch.as_deg()) <= 90.) {
```

Comment:
This removed code, might also hinder the work of OpenGL renderer, so you should use `ifdef`s here.